app/helpers/application_helper/button/auth_key_pair_cloud_create.rb

```diff
@@ -2,11 +2,7 @@ class ApplicationHelper::Button::AuthKeyPairCloudCreate < ApplicationHelper::But
   def disabled?
     # check that at least one EMS the user has access to supports
     # creation or import of key pairs.
-    Rbac.filtered(ManageIQ::Providers::CloudManager.all).each do |ems|
-      if ems.supports?(:auth_key_pair_create)
-        return false
-      end
-    end
+    Rbac.filtered(ManageIQ::Providers::CloudManager.select(&:supports_auth_key_pair))
```

Comment:
Something is missing here.  This just does a query but does nothing with it, so the code will always fall thought and say that the error message is present (and thus disabled).  The `return false` was lost.

Reply:
:+1: yeah thanks, fixed.  For some reason the `return false` in the block looked to me like it was being used for a `.select` not "return from out of this method" (clearly wasn't looking very hard at it :cry:)